tests/common.py

```diff
@@ -1,19 +1,20 @@
 from pathlib import Path
 import re
-from distutils.version import StrictVersion
+from packaging.version import Version
 import yaml
 from urllib.parse import urljoin
 
 
```

Comment:
Not really related to this file, just need a place to put this comment:

We need a test module like https://github.com/asdf-format/asdf-transform-schemas/blob/master/tests/test_integration.py, that will catch issues with the DirectoryResourceMapping setup.

Reply:
see 6f95534